src/components/flow/Flow.tsx

```diff
@@ -191,15 +190,6 @@ export class Flow extends React.Component<FlowStoreProps, {}> {
     this.Plumber.reset();
   }
 
-  public UNSAFE_componentWillUpdate(prevProps: FlowStoreProps): void {
-    if (
-      prevProps.editorState.activityInterval === this.props.editorState.activityInterval &&
-      this.props.editorState.activityInterval !== ACTIVITY_INTERVAL
-    ) {
-      this.props.mergeEditorState({ activityInterval: ACTIVITY_INTERVAL });
-    }
-  }
-
```

Comment:
This is addressing a react warning. This is moved to the save method so that components can't update themselves after render.